daemon/oci_linux.go

```diff
@@ -285,10 +285,7 @@ func WithNamespaces(daemon *Daemon, c *container.Container) coci.SpecOpts {
 					})
 				}
 			case networkMode.IsHost():
-				setNamespace(s, specs.LinuxNamespace{
-					Type: specs.NetworkNamespace,
-					Path: c.NetworkSettings.SandboxKey,
```

Comment:
Not for this PR, but we should probably look if that magic `default` SandboxKey is actually still serving any purpose, or if such containers should just not have a SandboxKey set.

(We may need to have a close look for Windows as well on that matter; perhaps there's some other rules there)